### PR TITLE
Data archiving structure

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/data-archiving/archive.md [hunk 1]
```diff
@@ -236,4 +236,4 @@ The metadata field `_archiveJob` is automatically created in your account and
 
 ## Audit ingestion job requests
 
-The [Audit Event Index](security/audit-event-index.md) provides events logs in JSON when ingestion jobs are created, completed, and deleted. 
+The [Audit Event Index](/docs/manage/security/audit-event-index.md) provides events logs in JSON when ingestion jobs are created, completed, and deleted.
```
Comment (kimsauce) [inline docs/manage/data-archiving/archive.md:239]:
```suggestion
The [Audit Event Index](/docs/manage/security/audit-event-index) provides events logs in JSON when ingestion jobs are created, completed, and deleted. 
```

File: docs/manage/data-archiving/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+slug: /manage/data-archiving
+title: Data Archiving
+description: Data Archiving allows you to archive selected data to an external server or Amazon S3.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/icons/operations/send-data.png')} alt="icon" width="50"/>
+
+Archive allows you to forward log data from Installed Collectors to AWS S3 buckets to collect at a later time. If you have logs that you don't need to search immediately you can archive them for later use. You can ingest from your Archive on-demand with five-minute granularity.
```
Comment (kimsauce) [inline docs/manage/data-archiving/index.md:11]:
```suggestion
Archive allows you to forward log data from Installed Collectors to AWS S3 buckets to collect at a later time. If you have logs that you don't need to search immediately, you can archive them for later use and ingest from your Archive on-demand with five-minute granularity.  
```

File: docs/manage/data-archiving/installed-collectors.md [hunk 3]
```diff
@@ -1,33 +1,33 @@
 ---
 id: installed-collectors
-title: Forward Data from an Installed Collector
-description: Learn how to set up Data Forwarding destinations for Installed Collectors.
+title: Archive Log Data to other destinations
```
Comment (kimsauce) [inline docs/manage/data-archiving/installed-collectors.md:3]: Doc titles are titlecase

```suggestion
title: Archive Log Data to Other Destinations
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
looks good, just minor stuff

File: docs/manage/data-archiving/index.md (hunk 2 above)
Comment (JainM6) [reply docs/manage/data-archiving/index.md:11]: Thanks @kimsauce I have slightly tweaked this sentence and split the last sentence.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**